i18n(zh-cn): translate crystallize.mdx,datocms.mdx,decap-cms.mdx,directus.mdx,frontmatter-cms.mdx (#3875)
withastro/docs: src/content/docs/zh-cn/guides/cms/directus.mdx

---
title: Directus & Astro
description: 使用Directus作为CMS向您的Astro项目添加内容
type: cms
stub: true
service: Directus
i18nReady: true
---

> [Directus](https://directus.io/)是一个后端即服务，可用于为您的Astro项目托管数据和内容。

Reviewer: ```suggestion
[Directus](https://directus.io/)是一个后端即服务，可用于为你的 Astro 项目托管数据和内容。

```

## 官方资源

> - 查看官方指南：[使用Directus开始构建Astro网站](https://directus.io/guides/get-started-building-an-astro-website-with-directus/)。

Reviewer: ```suggestion
- 查看官方指南：[使用 Directus 开始构建 Astro 网站](https://directus.io/guides/get-started-building-an-astro-website-with-directus/)。

```

> - Directus提供了一个[示例Astro博客模板](https://github.com/directus/examples/tree/main/astro)。

Reviewer: ```suggestion
- Directus 提供了一个[示例 Astro 博客模板](https://github.com/directus/examples/tree/main/astro)。

```

## 社区资源

> - 添加您的资源！

Reviewer: ```suggestion
- 添加你的资源！

```
